eslint.config.mjs

```diff
@@ -0,0 +1,67 @@
+import eslint from '@eslint/js';
+import simpleImportSort from 'eslint-plugin-simple-import-sort';
+import unusedImports from 'eslint-plugin-unused-imports';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+const todo_promoteToError = 'warn';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  {
+    plugins: {
+      'simple-import-sort': simpleImportSort,
+      'unused-imports': unusedImports,
+    },
+    rules: {
+      'simple-import-sort/imports': 'error',
+      'simple-import-sort/exports': 'error',
+      'unused-imports/no-unused-imports': 'error',
+    },
+    languageOptions: {
+      ecmaVersion: 2022,
+      sourceType: 'module',
+      globals: {
+        ...globals.browser,
+        ...globals.node,
+      },
+    },
+  },
+  {
+    files: ['**/*.ts', '**/*.tsx'],
+    // TODO: enable "TypeChecked" variants of these
+    // see https://typescript-eslint.io/getting-started/typed-linting#shared-configurations
+    extends: [...tseslint.configs.recommended, ...tseslint.configs.stylistic],
+    languageOptions: {
+      parserOptions: {
+        // TODO: enable type-aware checks
+        // indicates to find the closest tsconfig.json for each source file
+        // project: true,
+      },
+    },
+    rules: {
+      '@typescript-eslint/ban-ts-comment': todo_promoteToError,
```

Comment:
why todo? since the codebase is small, can some of them be errors already?

Reply:
Copy-pasta. I'll take a closer look.